csi-plug-n-play/sidecars/soda-proxy/pkg/controller/profile/profile.go

```diff
@@ -0,0 +1,29 @@
+package profile
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/sodafoundation/nbp/client/opensds"
+	"net/http"
+	"os"
+)
+
+func GetProfile(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	key := vars["id"]
+	opensdsEndpoint, errr := os.LookupEnv("OPENSDS_ENDPOINT")
+	if !errr {
+		fmt.Println("No env variables found for endpoint, switching to default")
```

Comment:
Why not use logger?

Reply:
Default logger will not work in this as the sodafoundation/api/client makes their own log-dir which creates a panic at runtime, will take this up in further iteration.

Reply:
Ok